docs/features/techdocs/creating-and-publishing.md

```diff
@@ -30,6 +30,11 @@ the documentation template.
 Create an entity from the documentation template and you will get the needed
 setup for free.
 
+!!! warning Currently the scaffolder plugin is limited to create repositories
```

Comment:
Please refer to this as the "Backstage Software Templates" since it is the official name of this feature.